src/ServiceDesk/RequestService.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+namespace JiraRestApi\ServiceDesk;
+
+use JiraRestApi\Configuration\ConfigurationInterface;
+use JiraRestApi\JiraClient;
+use JiraRestApi\Pagination\PaginatedQuery;
+use JiraRestApi\ServiceDesk\Attachment\Attachment;
+use JiraRestApi\ServiceDesk\Attachment\AttachmentCreationResult;
+use JiraRestApi\ServiceDesk\Attachment\AttachmentCreationResultInterface;
+use JiraRestApi\ServiceDesk\Attachment\AttachmentInterface;
+use JiraRestApi\ServiceDesk\Comment\Comment;
+use JiraRestApi\ServiceDesk\Comment\CommentInterface;
+use JiraRestApi\ServiceDesk\Date\Date;
+use JiraRestApi\ServiceDesk\Date\DateInterface;
+use JiraRestApi\ServiceDesk\Request\FieldValue;
+use JiraRestApi\ServiceDesk\Request\FieldValueInterface;
+use JiraRestApi\ServiceDesk\Request\Request;
+use JiraRestApi\ServiceDesk\Request\RequestInterface;
+use JiraRestApi\ServiceDesk\Request\Status;
+use JiraRestApi\ServiceDesk\Request\StatusInterface;
+use JiraRestApi\ServiceDesk\RequestType\Field;
+use JiraRestApi\ServiceDesk\RequestType\FieldInterface;
+use JiraRestApi\ServiceDesk\RequestType\FieldValue as RequestTypeFieldValue;
+use JiraRestApi\ServiceDesk\RequestType\FieldValueInterface as RequestTypeFieldValueInterface;
+use JiraRestApi\ServiceDesk\RequestType\RequestCreationMeta;
+use JiraRestApi\ServiceDesk\RequestType\RequestCreationMetaInterface;
+use JiraRestApi\ServiceDesk\RequestType\RequestType;
+use JiraRestApi\ServiceDesk\RequestType\RequestTypeInterface;
+use JiraRestApi\ServiceDesk\ServiceDesk\ServiceDesk;
+use JiraRestApi\ServiceDesk\ServiceDesk\ServiceDeskInterface;
+use JiraRestApi\ServiceDesk\User\User;
+use JiraRestApi\ServiceDesk\User\UserInterface;
+use JiraRestApi\ServiceDeskTrait;
+use Psr\Log\LoggerInterface;
+
+/**
+ * The interface for service desk request operations, matching the Request group in API.
+ * @link https://developer.atlassian.com/cloud/jira/service-desk/rest/api-group-request
+ */
+class RequestService extends JiraClient implements RequestServiceInterface
```

Comment:
Injecting more things may not be convenient for consumers, because there is no containers. 

And there was an issue about moving to PSR-18 linked in the description, probably it's better to keep all services as is and consistent until it is done.

Reply:
Understood and agreed. Let's hope that they accept a PR though, because then they'd have to maintain this.